http-handler.abi.md

```diff
@@ -0,0 +1,15 @@
+# Functions
+
+----
+
+#### <a href="#log" name="log"></a> `log` 
+
+  logs a message to the host's logs.
+  
+  Note: A host who fails to log the message will trap (aka panic,
+  "unreachable" instruction).
+##### Params
+
+- <a href="#log.message" name="log.message"></a> `message`: `u32`
+- <a href="#log.message_len" name="log.message_len"></a> `message-len`: `u32`
```

Comment:
here you can notice wit renaming the case format. I'm not sure the rationale of the input format being lower-hyphen, yet only emitting lower_snake. I'm arbitrarily using WASI's conventions as they literally wrote this tool.